src/OrchardCore/OrchardCore.Users.Abstractions/IdentityErrorCode.cs

```diff
@@ -0,0 +1,26 @@
+namespace OrchardCore.Users;
+
+public class IdentityErrorCode
+{
+    public const string PasswordRequiresDigit = nameof(PasswordRequiresDigit);
```

Comment:
I don't even agree with the technicality of these changes and using nameof everywhere because the day we decide to change the property name, even in casing, then it will break the underlying behavior or localization.

Reply:
Would be fine if it was just `public const string PasswordRequiresDigit = "PasswordRequiresDigit"` but we don't even need it, the string is not used anywhere else, no need to synchronize them.

Reply:
What about creating `private` class? If No, you can close the PR